voxblox/include/voxblox/io/layer_io.h

```diff
@@ -137,6 +137,66 @@ bool LoadBlocksFromFile(
   return true;
 }
 
+/*TODO(mereweth@jpl.nasa.gov) - best name for this function?
+ * This function may be useful when debugging a malformed protobuf dump
+ * Also prevents segfaulting Python using file_path EsdfMap constructor
+ */
+template <typename VoxelType>
+typename Layer<VoxelType>::Ptr LoadOrCreateLayerHeader(
```

Comment:
Ok! My next PR will propose replacing this with a better-engineered overload for LoadLayer, still for Python bindings. In the meantime, I'll add a more descriptive comment and a note about the upcoming PR.